### PR TITLE
Code refactor: Switch from BASH to Ansible playbooks as an deployment hypervisor

### PR DESCRIPTION
**Pull request by ArseniiPetrovich**
New features:
- [x] Upload config file to S3 bucket

Refactored:
- [x] Now Ansible manages S3 bucket and DynamoDB, and Terraform manages the rest infra
- [x] Installation splitted into easy-to-modify roles
- [x] Single point of configuration 
- [x] Readme updated

Important! This PR still doesn't reflect changes made in #86 PR, WIP. Also, we need to test this well including the edge cases before merging.

### CODE REVIEW

**Comment by ArseniiPetrovich** [comment]
@acravenho Can you briefly describe what `GRAPHIQL_TRANSACTION` variable is for? How we can generate/get value for it? I will add it at ReadMe. 
I believe `A map of chain names to random transaction hash on that chain` description is not really enough.

**Comment by acravenho** [comment]
@ArseniiPetrovich `GRAPHIQL_TRANSACTION` is an environment variable that takes a transaction hash from a network to provide a sample query in the GraphIQL Playground. 

Here is more info https://github.com/poanetwork/blockscout/issues/1417

**Comment by ArseniiPetrovich** [comment]
Thanks, @acravenho 
What are the next steps on get this merged? I think we should add Pavel, Igor and Victor here.

**Comment by vbaranov** [comment]
Also @ArseniiPetrovich please handle merging conflicts.

**Comment by ArseniiPetrovich** [comment]
@vbaranov Done. Merge conflicts solved.

**Comment by vbaranov** [comment]
@ArseniiPetrovich could you please return `db_name` to the config of terraform deployment? As far as I understand for now DB name matches the prefix, but actually, it shouldn't. They can differ from each other.

**Comment by ArseniiPetrovich** [comment]
Updated, @vbaranov. Please, make sure to update variables file as it changed much.

File: README.md [hunk 1]
```diff
@@ -1,216 +1,247 @@
...
+  chain_heart_beat_timeout = {
+    "mychain" = 30
+  }
+  chain_heart_command = {
+    "mychain" = "systemctl restart explorer.service"
+  }
+  chain_blockscout_version = {
+    "mychain" = "v1.3.0-beta"
+  }
+  chain_db_name = {
```
Comment (vbaranov) [inline README.md]: I don't see `chain_db_id` param in README

File: README.md [hunk 2]
```diff
@@ -1,216 +1,247 @@
...
-need to manually check for, and remove, any resources that were not able to be
-deleted for you. You can use the `bin/infra resources` command to list all ARNs
-that are tagged with the unique prefix you supplied to the installer, but not
-all AWS resources support tags, and so will not be listed. Here's a list of such
-resources I am aware of:
+**Note 2**: the actual name of your resource must include prefix that you will use in this deployment.
+Example:
+  Real resource: tf-poa
+  `prefix` variable: tf
+  `db_id` variable: poa
```
Comment (vbaranov) [inline README.md]: This param has another name after the last changes, right?

**Comment by ArseniiPetrovich** [comment]
@vbaranov  You are absolutely right. Updated.
